platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -207,19 +211,20 @@ public IdpEndpoint(String signaturePropertiesPath, String encryptionPropertiesPa
     }
 
     public void init() {
-        try (
+        try (//
```

Comment:
They are there to appease the formatter. Without them it globs this into a nasty unreadable chunk